src/components/quick-search/quick-search-global-style.scss

```diff
@@ -107,6 +107,7 @@
 }
 
 .c__modal__scroller:has(.quick-search-container),
+.c__modal__scroller:has(.no-padding),
 .c__modal__scroller:has(.noPadding) {
```

Comment:
In this case, perhaps replace the old noPadding classes with no-padding and delete this one?